src/sentry/grouping/ingest/seer.py

```diff
@@ -176,9 +179,24 @@ def _circuit_breaker_broken(event: Event, project: Project) -> bool:
     return circuit_broken
 
 
+def _has_empty_stacktrace_string(event: Event, stacktrace_string: str) -> bool:
+    if stacktrace_string == "":
+        metrics.incr(
+            "grouping.similarity.did_call_seer",
+            sample_rate=options.get("seer.similarity.metrics_sample_rate"),
+            tags={
+                "call_made": False,
+                "platform": event.platform if event.platform else "unknown",
+                "blocker": "empty-stacktrace-string",
```

Comment:
TODO: realized I will be double counting these metrics since I return an empty stacktrace for invalid cases and create a new blocker [here](https://github.com/getsentry/sentry/pull/80946/files#diff-e4319ca69185e2767c449c63dc4c937c3078e091b86c993fa505d7c73e978a4dR334)